pkg/operator/controller/controller_dns_configmap_test.go

```diff
@@ -174,6 +175,36 @@ func TestDesiredDNSConfigmap(t *testing.T) {
 			},
 			expectedCoreFile: mustLoadTestFile(t, "forwardplugin_tls"),
 		},
+		{
+			name: "Check the default cache settings",
+			dns: &operatorv1.DNS{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: DefaultDNSController,
+				},
+				Spec: operatorv1.DNSSpec{
+					Cache: operatorv1.DNSCache{
+						PositiveTTL: metav1.Duration{},
+						NegativeTTL: metav1.Duration{},
+					},
+				},
+			},
+			expectedCoreFile: mustLoadTestFile(t, "default_corefile_no_cache_configured"),
+		},
+		{
+			name: "Default Corefile with cache configured",
```

Comment:
Can you add one more test with something odd in the TTLs, like 0.5s.

Reply:
Added in my local changes.